src/api/app/models/bs_request.rb

```diff
@@ -273,19 +273,11 @@ def self.sourcediff_has_shown_attribute?(sourcediff)
   #               or :target_project
   # @return [Array]  Array of Request objects
   def self.with_open_reviews_for(attributes)
-    with_reviews(attributes).where(state: 'new').rewhere('bs_requests.state': 'review').map(&:bs_request)
+    Review.with_attributes(attributes).where(state: 'new').rewhere('bs_requests.state': 'review').map(&:bs_request)
```

Comment:
[nitpick] Is the rewhere really necessary? Don't see where the bs_request.state can be set?

Reply:
I also think that `rewhere('bs_requests.state': 'review')` should be dropped. Even more, `where(state: 'new')` should IMO be a `rewhere` (to ensure we return the new ones) or more elegant:

```Ruby
Review.with(attributes.merge(state: 'new')).map(&:bs_request)
```

:bowtie: